host/discoverd.go

```diff
@@ -105,3 +109,15 @@ func (d *DiscoverdManager) ConnectPeer(ips []string) error {
 	}
 	return err
 }
+
+func (d *DiscoverdManager) UpdateTags(tags map[string]string) error {
+	d.mtx.Lock()
+	defer d.mtx.Unlock()
+	for k, v := range tags {
+		d.inst.Meta[host.TagPrefix+k] = v
```

Comment:
Perhaps we just remove any that are empty?

We could have `flynn-host tags del foo` which calls `UpdateTags(map[string]string{"foo": ""})`, and here we just `delete(d.inst.Meta, host.TagPrefix+k)` if `v` is empty?